<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPracticWindow</class>
 <widget class="QMainWindow" name="AdminPracticWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администрирование модуля &quot;Практика&quot; | RayTracing | Alpha 1.5v</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_add_work">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Добавить лабораторную работу</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/add-work-button.png</normaloff>images/add-work-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_edit_work">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Редактировать лабораторную работу</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/edit-work-button.png</normaloff>images/edit-work-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_check_work">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Проверка лабораторных работ</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/check-work-button.png</normaloff>images/check-work-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_help">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Получить справку</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/help-button.png</normaloff>images/help-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_statistics">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Статистика по сданным работам</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/stats-users-button.png</normaloff>images/stats-users-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Данные</string>
    </property>
    <addaction name="action_list_works"/>
    <addaction name="action_current_statuses"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_manual"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_current_statuses">
   <property name="text">
    <string>Текущие статусы для работ</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="action_manual">
   <property name="text">
    <string>Руководство по проверке работ</string>
   </property>
  </action>
  <action name="action_list_works">
   <property name="text">
    <string>Текущий список работ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
